ask21-set.xml

```diff
@@ -368,6 +371,22 @@ See AUTHORS.txt for credits
             </winch>
         </hitches>
     </sim>
+    <limits>
+        <max-spoiler-extension-speed>135</max-spoiler-extension-speed>
```

Comment:
This is wrong, according to the flight manual, max spoiler extension speed is the same as Vne, 280km/h. On the other hand the max G force the aircraft can withstand is reduced with the airbrakes open.